lienflash/lienflash-fe
# Tm/testing

#### What’s this PR do?
- Improves testing for Jobs, JobDetails, and jobsReducer
- Changes file paths for lien eligible jobs

#### Where should the reviewer start?
- look at files for testing and their js files
- look at Header, App, Homepage, and Jobs to see file path changes

#### How should this be manually tested?
- in terminal, run npm test to see passing tests
- in browser, click on eligible and filed jobs to see how display changed

#### What are the relevant tickets?
#37 

#### Need Help/Support
- Can't get the back button test to work in JobDetails; not sure how to get it to pass
- Need to figure out the logic to clear the error message as soon as a user clicks away from the page that needed it.
